README.md

```diff
@@ -1,2 +1,24 @@
-# Module-1-First-Project
-This the Module 1, Day 2 Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Day 2 Project
+> Description the project.
+This is a day 2 project
+## Built With
+
+- Major languages: English
```

Comment:
[OPTIONAL]
What this expected of you was to list the programming languages used and not human language.
Have a read this [article](https://medium.com/@anaharris/human-languages-vs-programming-languages-c89410f13252) for a clear distinction between the two